runtests.py

```diff
@@ -279,7 +283,7 @@ def main(argv):
             bench_args.extend(['--bench', a])
 
         if not args.bench_compare:
-            cmd = ['asv', 'run', '-n', '-e', '--python=same'] + bench_args
+            cmd = ['asv', 'run', '-n', '-e', '--python=same'] + asv_cfpath + bench_args
```

Comment:
```suggestion
            cmd = ['asv', 'run', '-n', '-e', '--python=same'] + bench_args
```
we only need custom build options for bench_compare

Reply:
Yes indeed,  where is my mind?